### ISSUE TITLE
The train/vaild data scanning takes too long time.. how to deal with this!?

### ISSUE THREAD

**Issue by hwanython** [ultralytics/yolov5#6437]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and [discussions](https://github.com/ultralytics/yolov5/discussions) and found no similar questions.

### Question

Hi, I am an impressive YOLOv5. And I study, run it!

**But I have to use Colab environment!**

**Whenever I change hyperparameter, I have run "train.py".**

**However, the time of scanning data is so long..** 

**about 14000 images take 5 hours!!.. is it normal???**

**How to deal with this?**

**For instance, checkpoint the scanning and Resume only training.. like this..**

### Additional

![image](https://user-images.githubusercontent.com/86281561/151177515-8dcc13fb-aef1-48ad-abfc-46b34acdd01b.png)

**Comment by glenn-jocher** [comment]
@hwanython your dataset is being read over a network which is a terrible workflow. Always train on local data.

**Comment by hwanython** [comment]
Thank you for replying!! I got it.
And,, Is the way which change the loss function ?
I checked the utils/loss.py
the train.py code use "Compute Loss", If I want use "Focal Loss"..
How to do?

If my understanding is not correct, just saying please!!

Thank you!
